tests/unit/test_storage.py

```diff
@@ -132,6 +132,180 @@
     }
 """
 
+CEPH_OSD_DF_TREE = """
```

Comment:
I thought you don't want to modify them because the current one has been taken out of a real sosreport?

If copy this over to `sos_commands/ceph/json_output/ceph_osd_df_tree_--format_pretty`, it'll not match w/ its plain-text counterpart in `sos_commands/ceph/ceph_osd_df_tree`. If I replace both of them w/ from a real one, it may not be in sync w/ the rest of the stuff in sos_commands/ceph/ :D 

I can move this to tests/unit/fake_data_root if you prefer.

So I thought a string will not cause w/ any such issues (albeit being large & ugly). Perhaps we can move all these test-specific strings to another file and import it?